build-java/test-project/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Ignore Gradle project-specific cache directory
```

Comment:
idem as examples, I think test should be under main/test, not in a separate project

Reply:
(same response as before ;) )